reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,97 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that is shipped, installed, and configured separately from Windows PowerShell.
+Therefore, PowerShell Core is not included in the Windows 7/8.1/10 or Windows Server licensing agreements.
+
+However, PowerShell Core is supported under traditional Microsoft support agreements, including [Premier][], [Microsoft Enterprise Agreements][enterprise-agreement], and [Microsoft Software Assurance][assurance].
+You can also pay for [assisted support][] for PowerShell Core by filing a support request for your problem.
+
+We also offer [community support][] on GitHub where you can file an issue, bug, or feature request.
+Alternatively, you may find help from other members of the community on the general [Microsoft Community][] or the Microsoft [PowerShell Tech Community][].
+We offer no guarantee there that your issue will be addressed or resolved in a timely manner.
+If you have a problem that requires immediate attention,
+you should use the traditional, paid support options.
+
+## Lifecycle of PowerShell Core
+
+PowerShell Core is adopting the [Microsoft Modern Lifecycle Policy][modern].
+This support lifecycle is intended to keep customers up-to-date with the latest versions.
+
+The version 6.x branch of PowerShell Core will be updated approximately once every six months (e.g. 6.0, 6.1, 6.2, etc.)
+
+> [!IMPORTANT]
+> You must update within six months after each new minor version release to continue receiving support.
+
+For example, if PowerShell Core 6.1 is released on July 1st, 2018,
+you would be expected to update to PowerShell Core 6.1 by January 1st, 2019 to maintain support.
+
+![PowerShell Core branch lifecycle][lifecycle-chart]
+
+The Modern Lifecycle Policy also requires that Microsoft give customers 12 months notice before discontinuing support for a product (i.e. PowerShell Core).
+
+Eventually, we expect PowerShell Core will adopt the "long-term servicing" approach where we would require only servicing and security updates to stay in support on a specific branch/version of 6.x.
+
+## Supported platforms
+
+PowerShell Core is officially supported on the following platforms:
+
+* Windows 7, 8.1, and 10
```

Comment:
We might want to mention that we only support PowerShell on these as long as DotNet and supports them and refer to there support page.  https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md#windows  This is largely to capture that they stop supporting the product when the underlying OS is no longer supported.